docs/groovy-script/mods/pedestalcrafting/pedestal_crafting.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Pedestal Crafting"
+titleTemplate: "Pedestal Crafting | CleanroomMC"
+description: "Converts a variable number of input items into a single output item. The recipe can be customized with various particles."
+source_code_link: "https://github.com/Ender-Development/PedestalCrafting-Patched/blob/master/src/main/java/me/axieum/mcmod/pedestalcrafting/compat/groovyscript/PedestalCrafting.java"
+---
+
+# Pedestal Crafting (Pedestal Crafting)
+
+## Description
+
+Converts a variable number of input items into a single output item. The recipe can be customized with various particles.
```

Comment:
looking at the code, this would actually be 2+ inputs - variable would indicate 0 or 1 as viable options.